### ISSUE TITLE
Any plans to restore fused layer normalization

### ISSUE THREAD

**Issue by shkarupa-alex** [keras-team/keras#20003]
Keras v2 uses fused_batch_norm to efficiently do layer normalization https://github.com/keras-team/keras/blob/v2.15.0/keras/layers/normalization/layer_normalization.py#L328 
As far as i understand it was numerically stable with mixed_fp16 https://github.com/keras-team/keras/blob/v2.15.0/keras/layers/normalization/layer_normalization.py#L198

V3 implementation uses plain operators and always cast inputs to float32.
Moreover pytorch has it's own layer normalization implementation (which as i think should be faster too).

So are you planing to restore that old tricks to make layer normalization work more efficiently?
